btree/btree.c

```diff
@@ -826,46 +915,67 @@ struct node_op {
  * are set by the caller, and the function sets ->s_rec.
  */
 struct slot {
-	struct nd          *s_node;
-	int                 s_idx;
-	struct m0_btree_rec s_rec;
+	const struct nd     *s_node;
```

Comment:
Field name alignment, see coding-style.

Reply:
*Identifiers* should be aligned, that is, all "s_"-s should start at the same column.